# - Replaces the call to system-wide mpiexec by the mpiexec from EASYBUILD
#
# author: Jiri Furst <jiri.furst@gmail.com>
--- OpenCoarrays-2.2.0.orig/src/extensions/cafrun.in	2018-08-12 21:57:19.475509128 +0200
+++ OpenCoarrays-2.2.0/src/extensions/cafrun.in	2018-08-12 22:00:21.411583642 +0200
@@ -55,7 +55,7 @@
 #
 
 caf_version='@CAF_VERSION@'
-CAFRUN="@MPIEXEC@"
+CAFRUN=$EBROOTOPENMPI/bin/mpiexec
 if [[ "${CAFRUN}" == @*@ ]]; then
   CAFRUN=mpiexec
 fi
